.appveyor.yml

```diff
@@ -100,14 +95,9 @@ for:
 
       # Install latest gradle
       - sh: "sudo apt-get -y remove gradle"
-      - sh: "sudo apt-get install go-dep"
```

Comment:
With the version bump in SAM CLI, we might need to clean these as well in case we have some integration tests there.

Reply:
SAM CLI does not have Go Dep support, so there is no updates on SAM CLI at all.